docs/dev-guide/contrib-guides-cpp.md

```
@@ -130,31 +130,38 @@ returns a result containing an error code if unsuccessful or the object if succe
+1. Types
+  * This includes type aliases (`typedef`, `using`, `enum`), nested structs and classes, and
+    `friend` types.
+2. Static constants
+3. Factory methods
+  * Declarations of static methods that return an object of the class being defined.
+4. Static methods
+5. Static data members
+6. Constructors
+7. Operators
+8. Destructor
+9. Methods implementing InheritedClass
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 6; Actual: 9; Style: 1/2/3

(MD029, ol-prefix)